modules/mod_comment/mod_comment.erl

```diff
@@ -178,7 +212,7 @@ install_comment_table(false, Context) ->
         {"comment_rsc_created_key", "rsc_id, created"},
         {"comment_created_key", "created"}
     ],
-    [ z_db:q("create index "++Name++" on comment ("++Cols++")", Context) || {Name, Cols} <- Indices ],
+    [ z_db:q("create index " ++ Name ++ " on comment (" ++ Cols ++ ")", Context) || {Name, Cols} <- Indices ],
```

Comment:
Line 215 is too long:     [ z_db:q("create index " ++ Name ++ " on comment (" ++ Cols ++ ")", Context) || {Name, Cols} <- Indices ],.